plugins/generator-1.19.2/forge-1.19.2/templates/tool.java.ftl

```diff
@@ -89,6 +89,16 @@ public class ${name}Item extends ${data.toolType?replace("Spade", "Shovel")?repl
 		</#if>);
 	}
 
+	<#if data.toolType == "Shield" && data.repairItems?has_content>
+	@Override public boolean isValidRepairItem(ItemStack itemstack, ItemStack repairitem) {

```

Comment:
This part needs to use new system merged yesterday for tests to be happy

Reply:
Yeah, I noticed, waking up slowly today, sorry :sweat_smile: